packages/amplify-cli/src/extensions/amplify-helpers/push-resources.ts

```diff
@@ -78,8 +78,8 @@ export const pushResources = async (
 
   // no changes detected
   if (!hasChanges && !context.exeInfo.forcePush && !rebuild) {
+    context.usageData.stopCodePathTimer(ManuallyTimedCodePath.PUSH_TRANSFORM);
```

Comment:
Could this be solved by `try { } finally {}` ? Or some AOP technique? 
This will come back if somebody adds another exit point in this function.

Reply:
I didn't added a try finally because the stop is actually called here, looks that the push timer is split in 2 different methods, one is starting it, and the second one is stopping: https://github.com/aws-amplify/amplify-cli/blob/dev/packages/amplify-provider-awscloudformation/src/push-resources.ts#L276
cc: @edwardfoyle